.eslintrc.cjs

```diff
@@ -1,13 +1,22 @@
 /* eslint-env node */
 module.exports = {
-  ignorePatterns: ["src", "packages/*/dist"],
+  env: {
+    node: true,
+    // https://stackoverflow.com/a/60690811
+    // https://eslint.org/docs/latest/use/configure/language-options
+    es6: true,
+    browser: true,
+    commonjs: true
+  },
+  ignorePatterns: ["packages/*/dist"],
   extends: [
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
+    "plugin:choreography-lint/base", // Activate the custom rules
     "prettier",
   ],
   parser: "@typescript-eslint/parser",
-  plugins: ["@typescript-eslint"],
+  plugins: ["@typescript-eslint", "choreography-lint"],
```

Comment:
I don't think we want to enable the plugin project-wide. eslint config can be overwritten per directory.